src/acl/Makefile.am

```diff
@@ -150,7 +150,9 @@ libacls_la_SOURCES = \
 	Gadgets.cc \
 	Gadgets.h \
 	AclSizeLimit.cc \
-	AclSizeLimit.h
+	AclSizeLimit.h \
+	ConnMark.h \
```

Comment:
Please move these additions up to the C section below ConnectionsEncryptted.* entries. It also goes .cc before .h in the listing.